website/www/site/content/en/contribute/_index.md

```diff
@@ -127,173 +128,9 @@ script which is part of the Beam repo:
 1. Assign the issue to yourself. To get the permission to do so, email
    the [dev@ mailing list](/community/contact-us)
    to introduce yourself and to be added as a contributor in the Beam issue tracker including your
-   ASF Jira Username. For example [this welcome email](
-   https://lists.apache.org/thread.html/e6018c2aaf7dc7895091434295e5b0fafe192b975e3e3761fcf0cda7@%3Cdev.beam.apache.org%3E).
+   ASF Jira Username. For example [this welcome email](https://lists.apache.org/thread.html/e6018c2aaf7dc7895091434295e5b0fafe192b975e3e3761fcf0cda7@%3Cdev.beam.apache.org%3E).
 1. If your change is large or it is your first change, it is a good idea to
    [discuss it on the dev@ mailing list](/community/contact-us/).
 1. For large changes create a design doc
    ([template](https://s.apache.org/beam-design-doc-template),
    [examples](https://s.apache.org/beam-design-docs)) and email it to the [dev@ mailing list](/community/contact-us).
-
-### Development Setup {#development-setup}
-
-1. If you need help with git forking, cloning, branching, committing, pull requests, and squashing commits, see
-   [Git workflow tips](https://cwiki.apache.org/confluence/display/BEAM/Git+Tips)
-1. Familiarize yourself with gradle and the project structure. At the root of the git repository, run:
```

Comment:
Has this information moved somewhere else, or are we getting rid of it?

Reply:
we are getting rid of it

Reply:
I don't think that's a good idea. Before we remove a completele remove a section there should be a discussion on the mailing list to make sure the community agrees this section isn't valuable.

Could you leave it in for now? We can file a follow-up jira to consider removing it.

Reply:
Filed BEAM-11751 to maybe remove it in the future